app/boards/post/page.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+import styles from "./Page.module.css";
+import FileInput from "@/components/BoardPost/FileInput";
+import { useState } from "react";
+import { useRouter } from "next/navigation";
+import { BASE_URL } from "@/app/apis/getArticle";
+
+type Values = {
+  title: string;
+  content: string;
+  imgFile: File | null;
+};
+
+export default function PostPage() {
+  const router = useRouter();
+  const [values, setValues] = useState<Values>({
+    title: "",
+    content: "",
+    imgFile: null,
+  });
+
+  const handleChange = (name: keyof Values, value: string | File | null) => {
```

Comment:
이 부분은 제네릭을 활용한다면 조금 더 깔끔하게 만들어 낼 수 있을 듯 합니다~
아래처럼 가능할 듯 한데 한번 참고해보시고 이렇게 제네릭으로 만드는 연습을 해보시는 것도 좋을 듯 합니다.
```suggestion
  const handleChange = <T extends keyof Values>(name: T, value: Values[T]) => {
```